### PR TITLE
Typo fix to 'server basics' doc.

### PR DESCRIPTION
**Pull request by krisjey**
Removed non-existing selection annotations.
Fixing code errors (parentheses and braces).
Adding a missing import statement.

### CODE REVIEW

File: site/src/sphinx/server-basics.rst [hunk 1]
```diff
@@ -45,13 +48,26 @@ Even if we opened a port, it's of no use if we didn't bind any services to them.
 
 .. code-block:: java
 
-    import com.linecorp.armeria.common.HttpRequest;
-    import com.linecorp.armeria.common.HttpResponseWriter;
-    import com.linecorp.armeria.common.HttpStatus;
-    import com.linecorp.armeria.common.MediaType;
+    import static com.linecorp.armeria.common.SessionProtocol.HTTP;
```
Comment (minwoox) [inline site/src/sphinx/server-basics.rst]: We didn't put this import statement on purpose because it is already stated in the above example. :-)

**Comment by trustin** [comment]
@krisjey Gentle ping

**Comment by trustin** [comment]
@krisjey Geentle ping

**Comment by trustin** [comment]
Superceded by #1041

**Pull request closed without merging**